modules/react/button/stories/button/Button.stories.mdx

```diff
@@ -1,12 +1,5 @@
 import {Specifications, SymbolDoc, SymbolDescription} from '@workday/canvas-kit-docs';
 
-import {
-  PrimaryButton,
-  SecondaryButton,
-  TertiaryButton,
-  DeleteButton,
-} from '@workday/canvas-kit-react/button';
```

Comment:
These component imports are not used anywhere in the Buttons MDX (since they're not being used by the `Meta` SB component).